src/writer.rs

```diff
@@ -363,6 +363,20 @@ impl<'a, W> ElementWriter<'a, W> {
         self.start_tag.extend_attributes(attributes);
         self
     }
+
+    /// Push a new line inside an element.
+    pub fn element_new_line(mut self, indentation: usize) -> Self  {
+        if let Some(ref i) = self.writer.indent {
+            self.start_tag.push_newline();
+            let current_indent = i.current();
+            let mut indent = vec![i.indent_char; indentation];
+            indent.extend(current_indent);
+            self.start_tag.push_indent(&indent);
+        } else {
+            self.start_tag.push_newline();
+        }
```

Comment:
I think, that when indents are not enabled, this should be no-op
```suggestion
        }
```